.github/workflows/ci.yml

```diff
@@ -0,0 +1,187 @@
+name: Hedera Services CI
+
+on: [push]
+
+jobs:
+  Run-Unit-Tests:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_DB: fcfs
+          POSTGRES_PASSWORD: password
+          POSTGRES_USER: swirlds
+          PGDATA: /var/lib/postgresql/data/pgdata
+          POSTGRES_PORT: 5432
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'adopt'
+          cache: maven
+      - name: Build with Maven
+        run: mvn install
```

Comment:
When running maven in CI you should pass parameters like `--batch-mode` and `--no-transfer-progress`. Instead of adding them to every command, I recommend adding a `.mvn/maven.config` file with common maven parameters to standardize build arguments and stay DRY. Here's what the mirror node has:

```
--also-make --batch-mode --no-transfer-progress --show-version --threads 1C -Dmaven.artifact.threads=10 -Dmaven.wagon.provider.http=httpclient -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -Dmaven.wagon.http.retryHandler.count=5 -Dmaven.wagon.rto=10000
```

I'd recommend all these parameters except maybe `--threads 1C` if services doesn't support building modules in parallel.